src/main/java/seedu/duke/userinterface/command/Add.java

```diff
@@ -1,8 +1,48 @@
 package seedu.duke.userinterface.command;
 
+import seedu.duke.exceptions.IncorrectDeadlineFormatException;
+import seedu.duke.exceptions.TaskTitleException;
+import seedu.duke.exceptions.TaskWrongFormatException;
+import seedu.duke.tasks.Task;
+import seedu.duke.tasks.TaskList;
+import seedu.duke.userinterface.CliMessages;
+import seedu.duke.userinterface.InputParser;
+
 public class Add extends CliCommand {
+    public static final String COMMAND_WORD = "add";
+    public static final String TASK_DELIMITER = "/t";
+    public static final String DEADLINE_DELIMITER = "/by";
```

Comment:
We could have subclasses of CliCommand for commands with and without arguments, and the subclass with arguments could contain these fields to be overridden. Do you think that might work?

Reply:
Yea sure, that would be nice.

Reply:
I'll just add this as a `//TODO` for now